recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "gensim" %}
-{% set version = "4.1.2" %}
-{% set sha256 = "1932c257de4eccbb64cc40d46e8577a25f5f47b94b96019a969fb36150f11d15" %}
+{% set version = "4.2.0" %}
+{% set sha256 = "995ebd2970a31d47c100aaac10212f47e2bf12e2b06536d38883c951ff34eef1" %}
 
 package:
   name: {{ name|lower }}
```

Comment:
Could you please update `cython` pinning? See https://github.com/RaRe-Technologies/gensim/blob/acbba2f9562b90e0abed3a210bddb0fb20c7d619/setup.py#L321
And we can remove `dataclasses` from `run` and `morfessor` from `test/requires` https://github.com/RaRe-Technologies/gensim/commit/acce8a21377d1f599f8b9ec56b10321c91d4109d